resteasy-jaxrs/src/main/java/org/jboss/resteasy/core/ThreadLocalResteasyProviderFactory.java

```diff
@@ -557,6 +558,12 @@ public Configuration getConfiguration()
    }
 
    @Override
+   public Map<Class<?>, AsyncResponseProvider> getAsyncResponseProviders()
+   {
+      return getDelegate().getAsyncResponseProviders();
+   }
+
+   @Override

```

Comment:
Hi Stéphane,

This is nice. Thanks for working on it.

Do we need a

    public <T> AsyncResponseProvider<T> getAsyncResponseProvider(Class<T> type)

here?

-Ron